File: app/src/main/java/fr/clerc/myapplication/MetromobiliteService.java
package fr.clerc.myapplication;

import fr.clerc.myapplication.model.FeatureCollection;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MetromobiliteService {

    @GET("/api/findType/json")
    Call<FeatureCollection> getAgencyList(@Query("types") String type);

    @GET("/api/findType/json")
    Call<FeatureCollection> getAgency(@Query("types") String type, @Query("query") String name);
}

File: app/src/main/java/fr/clerc/myapplication/kotlin/MetromobiliteService.kt
package fr.clerc.myapplication.kotlin

import fr.clerc.myapplication.kotlin.model.FeatureCollection
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MetromobiliteService {

    @GET("/api/findType/json")
    fun getAgencyList(@Query("types") type: String): Call<FeatureCollection?>

    @GET("/api/findType/json")
    fun getAgency(@Query("types") type: String, @Query("query") name: String): Call<FeatureCollection?>
}